graylog2-web-interface/src/util/conditional/HideOnCloud.tsx

```diff
@@ -14,25 +14,17 @@
  * along with this program. If not, see
  * <http://www.mongodb.com/licensing/server-side-public-license>.
  */
-import pjson from '../../package.json';
+import * as React from 'react';
```

Comment:
It thinks it was renamed from `Version.js` to `HideOnCloud.tsx`, I guess because I lowered the threshold to detect renames. Since both `Version.ts` and `HideOnCloud.tsx` are there, I think it is safe to ignore.